### DISCUSSION TITLE
How to use a local repo?

### DISCUSSION THREAD

**Discussion by salkin-mada** [nvim-neorocks/rocks-git.nvim#33] [Q&A] [answered]
With `Packer` and `Lazy` I am used to quickly switch between remote and local git repositories. How do I do that with Rocks?
Could it be something like this maybe?
```lua
[plugins."something-fzf.nvim"]
path = "~/Code/Forks/something-fzf.nvim"
branch = "testing"
```
Or did I somehow miss a the completely obviously solution / way to do it.

**Comment by mrcjkb** [comment]
Hey 👋 

You can try [rocks-dev.nvim](https://github.com/nvim-neorocks/rocks-dev.nvim).

**Reply by salkin-mada** [reply, marked as answer]
With `rocks-dev.nvim`:
```lua
[plugins."fzf-something"]
dir = "~/Code/Forks/fzf-something"
```
